src/fidesops/service/connectors/saas_connector.py

```diff
@@ -278,5 +292,13 @@ def mask_data(
             rows_updated += 1
         return rows_updated
 
+    @staticmethod
+    def get_custom_function(function_location: str) -> Callable:
+        module, function_name = function_location.split(".")
+        module = importlib.import_module(
+            f"fidesops.service.saas_custom_functions.{module}"
```

Comment:
Might we want each function to live in its own file? Otherwise we're forced to grow an ever expanding file here which would get messy